src/components/ContentUploader/ContentUploader.js

```diff
@@ -251,116 +268,196 @@ class ContentUploader extends Component<Props, State> {
      *
      * @private
      * @param {Array<UploadFileWithAPIOptions | File>} files - Files to be added to upload queue
-     * @param {boolean} areAPIOptionsInFiles - whether file objects are UploadFileWithAPIOptions type
      * @param {Function} itemUpdateCallback - function to be invoked after items status are updated
      * @return {void}
      */
-    addFilesToUploadQueue = (
-        files: Array<UploadFileWithAPIOptions | File>,
-        areAPIOptionsInFiles,
-        itemUpdateCallback
-    ) => {
-        if (files.length <= 0) {
+    addFilesToUploadQueue = (files?: Array<UploadFileWithAPIOptions | File>, itemUpdateCallback: Function) => {
+        if (!files || files.length <= 0) {
             return;
         }
 
-        // $FlowFixMe webkitRelativePath is available on certain browsers
-        const isFolder = areAPIOptionsInFiles ? files[0].file.webkitRelativePath : files[0].webkitRelativePath;
-
         clearTimeout(this.resetItemsTimeout);
 
-        if (isFolder) {
-            this.addFilesWithRelativePathToQueue(files, areAPIOptionsInFiles, itemUpdateCallback);
+        const firstFile = getFile(files[0]);
+
+        // $FlowFixMe File type can have webkitRelativePath
```

Comment:
same comment as before, can get rid of few of these flow errors by appending webkitRelativePath to File?